assets/_locales/en/messages.json

```diff
@@ -3163,5 +3163,195 @@
   "select_running_dates": {
     "message": "Select running dates",
     "description": "Select running dates text"
+  },
```

Comment:
Missing CN translations for these. Any good AI or google translate can be used for these.